src/tests/unit/process-get-filters.test.js

```diff
@@ -0,0 +1,133 @@
+import chai from 'chai';
+
+import { GetFilterProcessor } from 'utils/process-get-filters';
+
+import { processGetFiltersData } from './test-data';
+
+chai.should();
+
+describe('normalizeFilterNames', () => {
+  const {
+    fooFilter, normalizedFooFilter,
+    nameFilter, normalizedNameFilter,
+  } = processGetFiltersData;
+  describe('when it gets an empty object', () => {
+    it('returns an empty object', () => {
+      const normalizedFilters = GetFilterProcessor.normalizeFilterNames({});
+      normalizedFilters.should.deep.equal({});
+    });
+  });
+  describe('when it gets a non empty object', () => {
+    it('changes filter keys but leaves non-filter keys unchanged', () => {
+      const normalizedFilters = GetFilterProcessor.normalizeFilterNames({
+        ...fooFilter,
+        ...nameFilter,
```

Comment:
I liked this approach for composing multiple test data objects together because I think it's pretty clear what it's doing and it lets me write less test data code overall. Open to suggestions though.